{"variants":[{"paths":["\/documentation\/dc\/walleterror\/underlyingerror(error:)"],"traits":[{"interfaceLanguage":"swift"}]}],"identifier":{"url":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/DC\/WalletError\/underlyingError(error:)","interfaceLanguage":"swift"},"schemaVersion":{"patch":0,"minor":3,"major":0},"sections":[],"metadata":{"role":"symbol","symbolKind":"case","modules":[{"name":"DC"}],"roleHeading":"Case","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"underlyingError","kind":"identifier"},{"kind":"text","text":"("},{"text":"error","kind":"externalParam"},{"text":": ","kind":"text"},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"text":")","kind":"text"}],"title":"WalletError.underlyingError(error:)","externalID":"s:2DC11WalletErrorO010underlyingC0yACs0C0_p_tcACmF"},"kind":"symbol","abstract":[{"type":"text","text":"A general registration error ocurred."}],"hierarchy":{"paths":[["doc:\/\/com.ibm.verify.ios.sdk\/documentation\/DC","doc:\/\/com.ibm.verify.ios.sdk\/documentation\/DC\/WalletError"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"kind":"keyword","text":"case"},{"kind":"text","text":" "},{"kind":"identifier","text":"underlyingError"},{"kind":"text","text":"("},{"text":"error","kind":"externalParam"},{"kind":"text","text":": "},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"text":")","kind":"text"}],"languages":["swift"],"platforms":["iOS"]}]}],"references":{"doc://com.ibm.verify.ios.sdk/documentation/DC/WalletError":{"role":"symbol","kind":"symbol","type":"topic","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"WalletError"}],"navigatorTitle":[{"kind":"identifier","text":"WalletError"}],"abstract":[{"type":"text","text":"A type that indicates when a wallet operation fails."}],"url":"\/documentation\/dc\/walleterror","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/DC\/WalletError","title":"WalletError"},"doc://com.ibm.verify.ios.sdk/documentation/DC/WalletError/underlyingError(error:)":{"type":"topic","kind":"symbol","url":"\/documentation\/dc\/walleterror\/underlyingerror(error:)","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/DC\/WalletError\/underlyingError(error:)","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"underlyingError","kind":"identifier"},{"kind":"text","text":"("},{"text":"error","kind":"externalParam"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP","text":"Error"},{"kind":"text","text":")"}],"title":"WalletError.underlyingError(error:)","abstract":[{"text":"A general registration error ocurred.","type":"text"}],"role":"symbol"},"doc://com.ibm.verify.ios.sdk/documentation/DC":{"type":"topic","identifier":"doc:\/\/com.ibm.verify.ios.sdk\/documentation\/DC","abstract":[],"kind":"symbol","title":"DC","role":"collection","url":"\/documentation\/dc"}}}